test/webhook_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+sed -ri "s|'(quay.io/reactiveops/polaris:).+'|'\1${CIRCLE_SHA1}'|" ./deploy/webhook.yaml
+kubectl apply &> /dev/null -f ./deploy/webhook.yaml
+sleep 15
+
```

Comment:
Instead of using `VAR1` and `VAR2`, I'd suggest setting a variable `PASSING=1` here (before any of the tests), then setting `PASSING=0` in the else statements if any of the tests fails. That'll make it easier for us to add more tests down the line.